.github/workflows/deploy.yml

```diff
@@ -0,0 +1,102 @@
+name: Reusable cloudflare deploy
+
+on:
+  workflow_call:
+    inputs:
+      custom_domain:
+        required: false
+        type: string
+      production_branch:
+        required: false
+        type: string
+      build_artifact_name:
+        required: false
+        type: string
+      build_artifact_run_id:
+        required: false
+        type: string
+    secrets:
+      CLOUDFLARE_API_TOKEN:
+        required: true
+      CLOUDFLARE_ACCOUNT_ID:
+        required: false
+      UBIQUITY_BOUNTY_BOT_APP_ID:
+        required: true
+      UBIQUITY_BOUNTY_BOT_PRIVATE_KEY:
+        required: true
+
+jobs:
+  deploy-to-cloudflare:
+    name: Deploy to Cloudflare Pages
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          repository: ubiquity/.github
```

Comment:
This workflow is implied to be used by other repos so shouldn't we delete this line `repository: ubiquity/.github` ?

Reply:
it needs util functions from `.github` repo, it doesn't need access to caller repo because it downloads the artifact